### PR TITLE
Add some improvements

### PR DESCRIPTION
**Pull request by genuss**
Despite of #2 I would like to propose some improvements in current ``syslog-ng.conf`` template.
Things made: parametrized ``@module`` and ``version``, added ``@include`` directive.
Also I kept backwards compatibility with current template.

### CODE REVIEW

File: syslog_ng/files/syslog-ng.conf [hunk 1]
```diff
@@ -16,9 +16,17 @@ def rule_builder(rule):
       return rule[1:]
     return '"%s"' % rule
 %>\
-@version: 3.3
 
-@module tfjson
+@version: ${ syslog_config.get('version', '3.3') }
+
+% for module in syslog_config.get('module', ['tfjson']):
```
Comment (puneetk) [inline syslog_ng/files/syslog-ng.conf]: Is `tfjson` a default module on all installs, or are we turning it on for everyone.

Comment (genuss) [inline syslog_ng/files/syslog-ng.conf]: Well, I didn't find it on Debian by default.
Looks like this formula turns it on for everyone.

Comment (puneetk) [inline syslog_ng/files/syslog-ng.conf]: Can you please change it from 
```
['tfjson']
```
to 
```
[]
```
We don't want an unintentional feature install for people.

Comment (genuss) [inline syslog_ng/files/syslog-ng.conf]: Completely agreed and made it.

**Pull request merged**